packages/lesswrong/components/jargon/GlossaryEditForm.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { Components, getFragment, registerComponent } from '../../lib/vulcan-lib';
+import { useMutation, gql } from '@apollo/client';
+import { useMulti } from '@/lib/crud/withMulti';
+import Button from '@material-ui/core/Button';
+
+const styles = (theme: ThemeType) => ({
+  root: {
+
+  },
+  generateButton: {
+    background: theme.palette.buttons.startReadingButtonBackground,
+
+    // TODO: Pick typography for this button. (This is just the typography that
+    // Material UI v0 happened to use.)
+    fontWeight: 500,
+    fontSize: "14px",
+    fontFamily: "Roboto, sans-serif",
+  },
+});
+
+export const GlossaryEditForm = (props: {
+  classes: ClassesType<typeof styles>,
+  document: PostsPage,
+}) => {
+  const { classes, document } = props;
+  const { JargonEditorRow, LoadMore, Loading } = Components;
+
+  const { results: glossary = [], loadMoreProps, refetch } = useMulti({
+    terms: {
+      view: "jargonTerms",
+      postId: document._id,
+      limit: 100
+    },
+    collectionName: "JargonTerms",
+    fragmentName: 'JargonTermsFragment',
+    ssr: true,
+  })
+
+  const sortedGlossary = [...glossary].sort((a, b) => {
+    const termA = a.term.toLowerCase();
+    const termB = b.term.toLowerCase();
+    if (termA < termB) return -1;
+    if (termA > termB) return 1;
+    return 0;
+  });
```

Comment:
Is this sorted here because there's no trivial way to do a case-insensitive lexicographic sort using a view?